Modules/Core/Common/include/itkImageBase.hxx

```diff
@@ -72,14 +72,13 @@ ImageBase<VImageDimension>::SetSpacing(const SpacingType & spacing)
 {
   for (unsigned int i = 0; i < VImageDimension; ++i)
   {
-    if (this->m_Spacing[i] < 0.0)
+    if (spacing[i] < 0.0)
     {
 #if !defined(ITK_LEGACY_REMOVE)
-      itkWarningMacro("Negative spacing is not supported and may result in undefined behavior. Spacing is "
-                      << this->m_Spacing);
+      itkWarningMacro("Negative spacing is not supported and may result in undefined behavior. Spacing is " << spacing);
```

Comment:
This message should say something like:

"Negative spacing is not supported and may result in undefined behavior. Proceeding to set spacing to " <<